docs/guidance/publication.md

```diff
@@ -136,7 +136,7 @@ libcoveofds jsontogeojson network-package.json nodes.geojson spans.geojson
 
 To convert data to CSV format:
 
-- [Install Flatten Tool](https://flatten-tool.readthedocs.io/en/latest/getting-started/#getting-started)
+- [Install Flatten Tool](https://flatten-tool.readthedocs.io/en/latest/getting-started.html)
```

Comment:
Sorry, this change shouldn't have happened, see https://github.com/OpenDataServices/flatten-tool/pull/423 to fix it.

Reply:
That fix is merged, and I've removed the commit that changed this link.